### PR TITLE
tweaks for continuous integration runs

### PR DESCRIPTION
**Pull request by rtertiaer**
### What does this change intend to accomplish?
This PR contains tweaks that permit running `./scripts/configure.py` (our installer) during continuous integration. This is foundational work for #471 , and in my test case produces an image now. It also runs cleanly against my test case AmpliPro Streamer on my desk.

I'm iterating on this a bit still, but I'm opening a draft PR now for any feedback.

### Checklist

* [x] Have you tested your changes and ensured they work?
* [x] Have you checked to ensure there aren't other open [Pull Requests](../../../pulls) for the same update/change?
* [x] If applicable, have you updated the CHANGELOG?
* [x] Does your submission pass linting & tests? You can test on localhost using `./scripts/test`

### CODE REVIEW

**Comment by rtertiaer** [comment]
This  is ready for review. The fancy [amplipi-img-gen](https://github.com/micro-nova/amplipi-img-gen) repo and this PR work in tandem to create full blown AmpliPi images. They work out of the box at the moment; i've tested every stream type and appliance type I have around, and it all worked correctly. I suspect there is still a bug or two hiding, but this is pretty robust as it stands.

definitely made my life harder merging the linted `configure.py` earlier today.

**Comment by rtertiaer** [comment]
This closes #471.

File: CHANGELOG.md [hunk 1]
```diff
@@ -11,7 +12,7 @@
 * Streams
   * Remove stop command (only accessable through API) from Pandora
   * Disconnect zones from sources when they are disabled
-* Updated Pianobar to fork of 2022.04.01
+  * Updated Pianobar to fork of 2022.04.01
```
Comment (klay2000) [inline CHANGELOG.md:15]: oops, my b

**Review by klay2000 — APPROVED** [review]
lgtm, as far as I understand it

**Comment by rtertiaer** [comment]
the rebase after #773 left some detritus here; fixing.

File: scripts/configure.py [hunk 2]
```diff
@@ -837,42 +855,46 @@ def _update_web(env: dict, restart_updater: bool, progress) -> List[Task]:
...
+    tasks += print_progress(_start_service('amplipi-updater', test_url='http://0.0.0.0:5001/update'))
+  elif not restart_updater and not env['is_ci']:
     # start a second updater service and check if it serves a url
     # this allow us to verify the update the updater probably works
-    tasks += print_progress(_create_service('amplipi-updater-test',
-                            _update_service(env['base_dir'], port=5002)))
-    tasks += print_progress(_start_service('amplipi-updater-test',
-                            test_url='http://0.0.0.0:5002/update'))
+    tasks += print_progress(_create_service('amplipi-updater-test', _update_service(env['base_dir'], port=5002), env))
+    tasks += print_progress(_start_service('amplipi-updater-test', test_url='http://0.0.0.0:5002/update'))
```
Comment (linknum23) [inline scripts/configure.py:889]: I think this reads a little better if the whole thing is wrapped in if not env['ci']

**Review by linknum23 — COMMENTED** [review]
LGTM

File: scripts/configure.py (hunk 2 above)
Comment (rtertiaer) [reply scripts/configure.py:889]: oh nice yea totally

**Pull request merged**